sim/priest/_item_sets_pve.go

```diff
@@ -10,444 +10,114 @@ import (
 	},
 })
 
 ///////////////////////////////////////////////////////////////////////////
-//                            SoD Phase 4 Item Sets
+//                            Classic Phase 3 Item Sets - BWL
 ///////////////////////////////////////////////////////////////////////////
 
-var ItemSetVestmentsOfTheVirtuous = core.NewItemSet(core.ItemSet{
-	Name: "Vestments of the Virtuous",
+var ItemSetVestmentsOfTranscendence = core.NewItemSet(core.ItemSet{
+	Name: "Vestments of Transcendence",
 	Bonuses: map[int32]core.ApplyEffect{
-		// Increases damage and healing done by magical spells and effects by up to 23.
-		2: func(agent core.Agent) {
-			c := agent.GetCharacter()
-			c.AddStat(stats.SpellPower, 23)
-		},
-		// Your spellcasts have a 6% chance to energize you for 300 mana.
-		4: func(agent core.Agent) {
+		// Allows 15% of your Mana regeneration to continue while casting.
+		3: func(agent core.Agent) {
 			c := agent.GetCharacter()
-			actionID := core.ActionID{SpellID: 450576}
-			manaMetrics := c.NewManaMetrics(actionID)
-
-			core.MakeProcTriggerAura(&c.Unit, core.ProcTrigger{
-				Name:       "S03 - Mana Proc on Cast - Vestments of the Devout",
-				Callback:   core.CallbackOnCastComplete,
-				ProcMask:   core.ProcMaskSpellDamage | core.ProcMaskSpellHealing,
-				ProcChance: 0.06,
-				Handler: func(sim *core.Simulation, spell *core.Spell, _ *core.SpellResult) {
-					if c.HasManaBar() {
-						c.AddMana(sim, 300, manaMetrics)
-					}
-				},
-			})
+			c.PseudoStats.SpiritRegenRateCasting += .15
 		},
-		// +8 All Resistances.
-		6: func(agent core.Agent) {
-			c := agent.GetCharacter()
-			c.AddResistances(8)
+		// When struck in melee there is a 50% chance you will Fade for 4 sec.
+		5: func(agent core.Agent) {
+			// Nothing to do
 		},
-		// +200 Armor.
+		// Your Greater Heals now have a heal over time component equivalent to a rank 5 Renew.
 		8: func(agent core.Agent) {
-			c := agent.GetCharacter()
-			c.AddStat(stats.Armor, 200)
+			// Nothing to do
 		},
 	},
 })
 
-var ItemSetDawnProphecy = core.NewItemSet(core.ItemSet{
-	Name: "Dawn Prophecy",
+///////////////////////////////////////////////////////////////////////////
+//                            Classic Phase 4 Item Sets - ZG and AB
+///////////////////////////////////////////////////////////////////////////
+
+var ItemSetConfessorsRaiment = core.NewItemSet(core.ItemSet{
+	Name: "Confessor's Raiment",
 	Bonuses: map[int32]core.ApplyEffect{
-		// -0.1 sec to the casting time of Flash Heal and -0.1 sec to the casting time of Greater Heal.
+		// Increases healing done by spells and effects by up to 22.
 		2: func(agent core.Agent) {
-			// Nothing to do
-		},
-		// Increases your critical strike chance with spells and attacks by 2%.
-		4: func(agent core.Agent) {
 			c := agent.GetCharacter()
-			c.AddStats(stats.Stats{
-				stats.MeleeCrit: 2 * core.CritRatingPerCritChance,
-				stats.SpellCrit: 2 * core.CritRatingPerCritChance,
-			})
+			c.AddStat(stats.SpellPower, 22)
```

Comment:
Since this is healing specifically you want want to use `stats.HealingPower` here